src/dash-adapter.js

```diff
@@ -403,6 +408,43 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
     }
   }
 
+  /**
+   * attach media - return the media source to handle the video tag
+   * @public
+   * @param {boolean} playbackEnded don't seek to the last detach point
+   * @returns {void}
+   */
+  attachMediaSource(playbackEnded: ?boolean): void {
+    if (!this._shaka) {
+      if (this._videoElement && this._videoElement.src) {
+        Utils.Dom.setAttribute(this._videoElement, 'src', '');
+        Utils.Dom.removeAttribute(this._videoElement, 'src');
```

Comment:
???

Reply:
Case that ima set empty source and it raise an error from video tag

Reply:
It could happened when IMA doesn't clear the src on ADBREAKEND or didn't finish the clear